int32 realVel
float32 realPos
float32 realCur
float32 volt1
float32 volt2
float32 torque
float32 cur_node1
float32 cur_node2

